packages/expo-image-picker/ios/MediaHandler.swift

```diff
@@ -26,13 +28,16 @@ internal struct MediaHandler {
     return try await concurrentMap(selection) { selectedItem in
       let itemProvider = selectedItem.itemProvider
 
+      if itemProvider.hasItemConformingToTypeIdentifier(Self.livePhotoTypeIdentifier) && options.useLivePhotos {
```

Comment:
Maybe we could use `canLoadObject(ofClass: PHLivePhoto.self)` instead? Then you don't need `livePhotoTypeIdentifier`